react/CartWrapper.tsx

```diff
@@ -32,6 +33,30 @@ const CartWrapper: FunctionComponent = () => {
 
   const cartEventFiredRef = useRef(false)
 
+  const { logKpiEvent } = useSplunk()
+
+  const prevOrderFormId = useRef<string | null>(null)
+
+  useEffect(() => {
+    if (loading || prevOrderFormId.current === orderForm.id) {
+      return
+    }
+
+    prevOrderFormId.current = orderForm.id
```

Comment:
nit: Although I don't really see how someone would not realize `prev` is short for `previous`, I'd rather be explicit about it. Agree?

```suggestion
  const previousOrderFormId = useRef<string | null>(null)

  useEffect(() => {
    if (loading || previousOrderFormId.current === orderForm.id) {
      return
    }

    previousOrderFormId.current = orderForm.id
```